src/types.deno.ts

```diff
@@ -115,18 +115,18 @@ export class InputFile {
                 );
             }
             const file = await Deno.open(data);
-            return iterateReader(file);
+            return file.readable[Symbol.asyncIterator]();
```

Comment:
Deno has already closed the file when the reader returns `{ done: true }`.

Maybe this is better :)